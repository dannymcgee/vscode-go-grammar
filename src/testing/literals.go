var x := "Foo"
var x := "Foo \"Bar\" \\"

var x := 'x'
var x := '\\'
var x := '\''
var x := '\a'
var x := '\b'
