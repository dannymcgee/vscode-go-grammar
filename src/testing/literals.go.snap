// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`literals should work 1`] = `

-> var x := "Foo"
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.interpreted.go
 |           ^^^ string.interpreted.go
 |              ^ punctuation.definition.string.end.go  string.interpreted.go
-> var x := "Foo \\"Bar\\" \\\\"
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.interpreted.go
 |           ^^^^ string.interpreted.go
 |               ^^ constant.character.escape.go  string.interpreted.go
 |                 ^^^ string.interpreted.go
 |                    ^^ constant.character.escape.go  string.interpreted.go
 |                      ^ string.interpreted.go
 |                       ^^ string.interpreted.go
 |                         ^ punctuation.definition.string.end.go  string.interpreted.go
-> 
-> var x := 'x'
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.rune.go
 |           ^ string.rune.go
 |            ^ punctuation.definition.string.end.go  string.rune.go
-> var x := '\\\\'
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.rune.go
 |           ^^ constant.character.escape.go  string.rune.go
 |             ^ punctuation.definition.string.end.go  string.rune.go
-> var x := '\\''
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.rune.go
 |           ^^ constant.character.escape.go  string.rune.go
 |             ^ punctuation.definition.string.end.go  string.rune.go
-> var x := '\\a'
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.rune.go
 |           ^^ constant.character.escape.go  string.rune.go
 |             ^ punctuation.definition.string.end.go  string.rune.go
-> var x := '\\b'
 | ^^^ storage.type.var.go
 |     ^ variable.other.go
 |       ^^ keyword.operator.assignment.go
 |          ^ punctuation.definition.string.begin.go  string.rune.go
 |           ^^ constant.character.escape.go  string.rune.go
 |             ^ punctuation.definition.string.end.go  string.rune.go
-> 
`;
